Fix refresh-button not hiding when upload was successful

moritzmenzel opened the pull request:
## Refresh button not hiding when upload was successful
I do not have the reason for that yet, but I figured out that the refresh-button (paper-icon-button to re-upload the file) won't hide even if it should when the upload of that file was successful. (Why would you want to re-upload an already uploaded file?).

The hidden$ - Attribute on the paper-icon-button somehow doesn't seem to work for me, wrapping it in a &lt;div&gt; with the attribute on it, it will behave correctly (Show when an error appears, hide when the upload was successful)

---

```HTML
<div hidden$="[[!file.error]]">
  <paper-icon-button ... ></paper-icon-button>
</div>
```

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/vaadin/vaadin-upload/155)
<!-- Reviewable:end -->

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/vaadin/vaadin-upload?pullRequest=155) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/vaadin/vaadin-upload?pullRequest=155) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/vaadin/vaadin-upload?pullRequest=155) it.</sub>

moritzmenzel commented:
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-upload/155)*
<!-- Sent from Reviewable.io -->

Saulis commented:
Hi!

Thanks for the PR!

Could you try if instead of adding the wrapping div, you add an explicit style for the hiding:

```css
paper-icon-button[hidden] {
  display: none;
}
```

moritzmenzel commented:
Hi @Saulis ,

Adding the explicit style for hiding actually worked fine. I already committed the change where I removed the wrapping div and added the code you suggested to the element's CSS.

moritzmenzel commented:
Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-upload/155)*
<!-- Sent from Reviewable.io -->

Saulis commented:
Thanks! We'll release this asap
